### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new section to Turn off the builtin feature

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6954

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -67,9 +67,17 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
...
 
+### Turn off the built-in reporting experience
 
-> [!IMPORTANT]
-> We don't recommend the built-in reporting experience in Outlook because it can't use the [user submission policy](./user-submission.md). We recommend using the Report Message add-in or the Report Phishing add-in instead.
+We don't recommend the built-in reporting experience in Outlook because it can't use the [user submission policy](./user-submission.md). We recommend using the Report Message add-in or the Report Phishing add-in instead.
+
+You need to be assigned permissions before you can run this cmdlet.  To find the permissions required to run any cmdlet or parameter in your organization, see [Find the permissions required to run any Exchange cmdlet](/powershell/exchange/find-exchange-cmdlet-permissions).
+
+Below is the powershell command to dsable the built-in reporting experience
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
Run the following PowerShell command to disable the built-in reporting experience:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 2]
```diff
@@ -67,9 +67,17 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
 
 - Organizations that have a URL filtering or security solution (such as a proxy and/or firewall) in place, must have ipagave.azurewebsites.net and outlook.office.com endpoints allowed to be reached on HTTPS protocol.
 
+### Turn off the built-in reporting experience
 
-> [!IMPORTANT]
-> We don't recommend the built-in reporting experience in Outlook because it can't use the [user submission policy](./user-submission.md). We recommend using the Report Message add-in or the Report Phishing add-in instead.
+We don't recommend the built-in reporting experience in Outlook because it can't use the [user submission policy](./user-submission.md). We recommend using the Report Message add-in or the Report Phishing add-in instead.
+
+You need to be assigned permissions before you can run this cmdlet.  To find the permissions required to run any cmdlet or parameter in your organization, see [Find the permissions required to run any Exchange cmdlet](/powershell/exchange/find-exchange-cmdlet-permissions).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
You need to be assigned permissions before you can run this cmdlet. To find the permissions required to run any cmdlet or parameter in your organization, see [Find the permissions required to run any Exchange cmdlet](/powershell/exchange/find-exchange-cmdlet-permissions).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**